sparkles/tests/test_checks.py

```diff
@@ -321,19 +321,47 @@ def test_guide_count_or():
         {'text': 'OR count of guide stars 3.65 < 4.0', 'category': 'critical'},
         {'text': 'OR with 4 guides but 5 were requested', 'category': 'caution'}]
 
-    # Set a scenario with guide_count in the 3.5 to 4.0 range and *but* with
-    # a creep away (maneuver angle <= 5), and confirm that is just a warning
-    # (not critical).
+
+def test_dither_low_guide_count():
+
+    # Set a scenario with guide_count in the 3.5 to 4.0 range
+    # dither == 4x4 to get no warnings
     stars = StarsTable.empty()
     stars.add_fake_constellation(n_stars=5, mag=[7.0, 7.0, 7.0, 10.2, 10.3])
-    aca = get_aca_catalog(**mod_std_info(n_fid=3, n_guide=5, obsid=1, man_angle_next=5),
+    aca = get_aca_catalog(**mod_std_info(n_fid=3, n_guide=5, obsid=1,
+                                         dyn_bgd_n_faint=0, dither=(4, 4)),
                           stars=stars, dark=DARK40,
                           raise_exc=True)
     aca = ACAReviewTable(aca)
-    aca.check_guide_count()
+    aca.check_dither()
+    assert len(aca.messages) == 0
```

Comment:
Won't there be a warning if guide_count < 4.0 with #182 merged? Maybe this PR should be based off that branch?

Reply:
In any case it would be good to add an assertion that the guide count is really in the expected range since the fractional guide count definition could change in the future.